packages/ws-client/src/data-source.ts

```diff
@@ -65,7 +65,7 @@ export function createWSDataSource(wsUrl: string): WebSocketDataSource {
 
   rws.addEventListener('open', () => {
     const keys = [...subscriptions.entries()]
-      .filter(([, handlers]) => handlers.size > 0)
+      .filter(([key, handlers]) => !!key && handlers.size > 0)
```

Comment:
Can key in a map be undefined? Please explain!

Reply:
I don't remember, probably to make ts happy

Reply:
oh I remember, key was null here for some reason and the server was breaking. we should find the root cause